clang/utils/TableGen/TableGen.cpp

```diff
@@ -500,6 +506,12 @@ bool ClangTableGenMain(raw_ostream &OS, RecordKeeper &Records) {
   case GenArmSmeRangeChecks:
     EmitSmeRangeChecks(Records, OS);
     break;
+  case GenArmSmeStreamingAttrs:
```

Comment:
We also need to do this for SVE (you seem to have added the logic for it in SveEmitter.cpp, but are not using it otherwise).